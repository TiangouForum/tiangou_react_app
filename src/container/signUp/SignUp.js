import React, { Component } from 'react'
import SignUpForm from '../../componets/SignUpForm/SignUpForm'
class SignUp extends Component {
  render () {
    return (
      <div>
        <SignUpForm formType='signUp' url='http://localhost:3000/signup'></SignUpForm>
      </div>
    )
  }
}

export default SignUp
