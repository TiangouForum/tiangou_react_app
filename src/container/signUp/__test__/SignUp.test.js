import React from "react";
import SignUp from "../SignUp";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const div = renderer.create(<SignUp/>).toJSON();
  expect(div).toMatchSnapshot();
});
