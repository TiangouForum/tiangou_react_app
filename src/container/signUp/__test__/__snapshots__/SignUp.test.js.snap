// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <form
    className="form-horizontal"
  >
    <div
      className="form-group"
    >
      <label
        className="col-sm-2 col-sm-offset-3 control-label"
        htmlFor="formHorizontalEmail"
      >
        username
      </label>
      <div
        className="col-sm-3"
      >
        <input
          className="form-control"
          id="formHorizontalEmail"
          onChange={[Function]}
          placeholder="username"
          type="string"
        />
      </div>
    </div>
    <div
      className="col-sm-3 col-sm-offset-5"
    >
      <div
        className="alert alert-danger"
        role="alert"
        style={
          Object {
            "display": "none",
          }
        }
      >
                     
        
                   
      </div>
    </div>
    <div
      className="form-group"
    >
      <label
        className="col-sm-2 col-sm-offset-3 control-label"
        htmlFor="formHorizontalPassword"
      >
        Password
      </label>
      <div
        className="col-sm-3"
      >
        <input
          className="form-control"
          id="formHorizontalPassword"
          onChange={[Function]}
          placeholder="password"
          type="password"
        />
      </div>
    </div>
    <div
      className="col-sm-3 col-sm-offset-5"
    >
      <div
        className="alert alert-danger"
        role="alert"
        style={
          Object {
            "display": "none",
          }
        }
      >
                     
        
                   
      </div>
    </div>
    <div
      className="form-group"
    >
      <label
        className="col-sm-2 col-sm-offset-3 control-label"
        htmlFor="formHorizontalRepeatPassword"
      >
        Repeat Password
      </label>
      <div
        className="col-sm-3"
      >
        <input
          className="form-control"
          id="formHorizontalRepeatPassword"
          onChange={[Function]}
          placeholder="repeat password"
          type="password"
        />
      </div>
    </div>
    <div
      className="col-sm-3 col-sm-offset-5"
    >
      <div
        className="alert alert-danger"
        role="alert"
        style={
          Object {
            "display": "none",
          }
        }
      >
                     
        
                   
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="col-sm-10 col-sm-offset-6"
      >
        <button
          className="btn btn-default"
          disabled={false}
          onClick={[Function]}
          type="submit"
        >
          Sign Up
        </button>
      </div>
    </div>
  </form>
</div>
`;
